### ISSUE TITLE
open special app on notify click

### ISSUE THREAD

**Issue by hallerdavid** [androidseb25/iGotify-Notification-Assistent#104]
Hello! Is it possible (or possible to add the feature) that i can open a special app on clicking a special notification?

For example -> someone rings at my door -> Loxone sends notification to igotify -> i click the notification -> Loxone App should open.

I think hopefully its possible, but it is apple.... so i think its not. :-/ 

Maybe there is an alternative way (button on notification, ..)

**Comment by androidseb25** [comment]
It might be possible, but the feature misses the intended purpose of the app, which is to securely notify rather than redirect to other apps. I’ll think about the feature request and will mark the issue as LTFR for now.

**Comment by hallerdavid** [comment]
Yes! I understand that its not the intended purpose - i just thought about redirecting all notifications to gotify for my smarthome (washing machine finished, doorbell, ...) - so i think gotify is also usefull for this (because... it is for notifications :) ) - the thing is, for example loxone is not really good in sending notifications, some people get the notifications, other not. 
So i want to find a better solution. But when i now tell my wife that she have to open Loxone manually, the WAF (Wife Acceptence Factor) is not high! ;-) 

So i think its a really cool (optional) feature to open a special app when there is a notification from tool X - so why not implement it when there is time :-) ? Users who don´t need it don´t have to use it 😇 i would really like! 

But for the moment, i will set iGotify up for the notifications what don´t need a opened app after seeing it. :-) Thank you for this awesome work (i really like it!!)

**Comment by androidseb25** [comment]
Feature will be in the update 1.4.2

**Comment by androidseb25** [comment]
released issue now can be closed
